frame/contracts/src/benchmarking/mod.rs

```diff
@@ -877,7 +877,6 @@ benchmarks! {
 		let origin = RawOrigin::Signed(instance.caller.clone());
 	}: call(origin, instance.addr, 0u32.into(), Weight::MAX, None, vec![])
 
-	#[skip_meta]
```

Comment:
Why change an existing benchmark?

Reply:
this #[skip_meta] relates to the seal_set_code_hash, probably github diffing messed it up. It relates to [6f925a9e64924f4152f8fa5fcf4fd9c2b8373794](https://github.com/paritytech/substrate/pull/10690/commits/6f925a9e64924f4152f8fa5fcf4fd9c2b8373794)

Reply:
Wrong. You PR removes it from `seal_set_storage_per_new_kb`. Manually compared your branch and master.